### PR TITLE
Darkside interrupt server

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

File: e2e/darkside_interrupt_sync.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+describe('Interrupt sync', () => {
+  it('should maintain correct balance when sync is interrupted', async () => {
+    await loadDarksideWallet();
+    await sleep(10000);
+
+    // verify pool balances
+    await waitFor(element(by.id('header.drawmenu'))).toBeVisible().withTimeout(sync_timeout);
+    await element(by.id('header.drawmenu')).tap();
+    await element(by.id('menu.fund-pools')).tap();
```
Comment (juanky201271) [inline e2e/darkside_interrupt_sync.test.js:79]: change for `menu.syncreport`

File: e2e/darkside_interrupt_sync.test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    await expect(element(by.id('orchard-total-balance.big-part'))).toHaveText(' 1.0000');
```
Comment (juanky201271) [inline e2e/darkside_interrupt_sync.test.js]: for 100_000 blocks...
use something like this:
```
await waitFor(async () => {
  const notyetsynced = Number(await element(by.id('syncreport.notyetsynced')).getAttributes('text'));
  return notyetsynced < 90_000
}).withTimeout(big number);

(here you have to close & open the App, and go to the scrren...)

await waitFor(async () => {
  const notyetsynced = Number(await element(by.id('syncreport.notyetsynced')).getAttributes('text'));
  return notyetsynced < 80_000
}).withTimeout(big number);

**Comment by Oscar-Pepper** [comment]
closed for #572

**Comment by juanky201271** [comment]
closed for #572

**Comment by juanky201271** [comment]
closed for https://github.com/zingolabs/zingo-mobile/pull/572

**Pull request closed without merging**